### PR TITLE
Added basic DB support

### PR DESCRIPTION
**Pull request by ZeyadTamimi**
Added support for PSQL using FLASK SQL Alchemy and set up initial tables

Also updated settings and pylintrc

Created scripts to setup PSQL automatically

### CODE REVIEW

File: scripts/psql_setup_linux [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Initialize the service
+sudo systemctl start postgresql.service
+sudo systemctl enable postgresql.service
+
+# Set up tables used by KAOS
+$PSQL_CMD createuser --interactive -P -s $DATABASE_USER
+$PSQL_CMD createdb $DATABASE_NAME
+
+echo Created database: $DATABASE_NAME and user $DATABASE_USER please updated kaos/settings.py
```
Comment (lsav) [inline scripts/psql_setup_linux]: typo -- I think by "please updated" you mean "please update"?

File: kaos/models.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        name:               The name of the satellite
+        timeposvel_records: Satellite ephemeris records
+    """
+    __tablename__ = 'SatelliteInfo'
+
+    uid = DB.Column(DB.Integer, primary_key=True)
+    name = DB.Column(DB.String(50), nullable=False)
+    timeposvel_records = DB.relationship("TimePosVel", backref='satellite_info', lazy=True)
+
+class SatelliteTimPosVel(DB.Model):
```
Comment (lsav) [inline kaos/models.py]: This name is a bit of a mouthful. Can we call it something simpler like "SatelliteOrbit" or "OrbitInfo" or even just "Orbit" instead?

File: kaos/models.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    """This table stores satellite ephemeris records at specific points in time.
+
+    The table holds the following information:
+        uid:            Unique ID for the particular record
+        satellite_uid   Unique ID for the satellite that owns this record
```
Comment (lsav) [inline kaos/models.py]: Since they use "PlatformID", might it be better to stick with that?

File: kaos/__init__.py [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
-"""
+"""KAOS application initialization code."""
 
 from flask import Flask
 
 def create_app():
-    """
-    Create and setup the KAOS app.
-    """
+    """Create and setup the KAOS app."""
```
Comment (lsav) [inline kaos/__init__.py]: Thanks for fixing docstrings!

File: .pylintrc [hunk 5]
```diff
@@ -16,8 +17,7 @@ score=no
 good-names=i, j, k, ex, Run, _, w, x, y, z
 
 # Maximum number of characters on a single line.
-max-line-length=80
+max-line-length=100
```
Comment (lsav) [inline .pylintrc]: :'(

Comment (lsav) [reply .pylintrc]: Also, should update README since it still says 80 chars.

File: kaos/models.py (hunk 2 above)
Comment (ZeyadTamimi) [reply kaos/models.py]: Sure

File: scripts/psql_setup_linux (hunk 1 above)
Comment (ZeyadTamimi) [reply scripts/psql_setup_linux]: lol, yeah.

**Pull request merged**